### DISCUSSION TITLE
RecordCount

### DISCUSSION THREAD

**Discussion by dantedantas** [AlexaCRM/dynamics-webapi-toolkit#85] [General]
Hello,

Please, could someone tell me how could I get the row amount for a query? I want to know how much rows my query found or if didnt found nothing.

![image](https://user-images.githubusercontent.com/10854571/190401711-3c6f741c-1a1b-422f-b029-b72b09ec01fc.png)

Thank you.
BR
Dante

**Comment by georged** [comment]
Hi Dante,

add `returntotalrecordcount` attribute to your FetchXml. That will include @odata.count value in the $result (but I'm not sure if we bubble it up correctly to the entity collection).

Thanks
George
